Examples/Python/src/TrackFitting.cpp

```diff
@@ -106,12 +107,17 @@ void addTrackFitting(Context& ctx) {
         .value("KLDistance", MixtureReductionAlgorithm::KLDistance);
 
     py::class_<ActsExamples::BetheHeitlerApprox>(mex, "AtlasBetheHeitlerApprox")
-        .def_static("loadFromFiles",
-                    &ActsExamples::BetheHeitlerApprox::loadFromFiles,
-                    py::arg("lowParametersPath"), py::arg("highParametersPath"),
-                    py::arg("lowLimit") = 0.1, py::arg("highLimit") = 0.2)
-        .def_static("makeDefault",
-                    []() { return Acts::makeDefaultBetheHeitlerApprox(); });
+        .def_static(
+            "loadFromFiles", &ActsExamples::BetheHeitlerApprox::loadFromFiles,
+            "lowParametersPath"_a, "highParametersPath"_a, "lowLimit"_a = 0.1,
+            "highLimit"_a = 0.2, "clampToRange"_a = false)
+        .def_static(
+            "makeDefault",
+            [](bool clampToRange) {
+              return Acts::makeDefaultBetheHeitlerApprox(clampToRange);
```

Comment:
Just a question: why bind this as a scoped static function instead of a free function like it is in C++? Or alternatively, why not make the free function a scoped static like in Python?

Reply:
The main reason is, that the C++ class `Acts::AtlasBetheHeitlerApproximation` is templated, and for the python binding a specific choice of template parameters is choosen. So in contrary to the C++ case it makes sense to have it as a static factory as part of the class.
But yeah, the differences between C++ and binding are also not optimal.
Given that the CI is green now I would like to keep it for now...